packages/ui/src/ui/store/actions/settings/index.ts

```diff
@@ -37,7 +38,7 @@ export function setSettingByKey<T>(path: SettingKey, value: T): SettingsThunkAct
     return (dispatch, getState) => {
         const {
             settings: {provider, data},
-            global: {login},
+            global: {login, cluster = YT.cluster},
```

Comment:
```ts
global: {login, cluster = getConfig().userSettingsCluster},
```
it is better to have selector for settings cluster:
```
const getSettingCluster = createSelector([getCluster], (cluster) => {
    return getConfig().userSettingsCluster ?? cluster;
})
```
and use it in all settings-related actions